downloadIstio.sh

```diff
@@ -18,9 +18,17 @@ else
   # TODO we should check more/complain if not likely to work, etc...
   OSEXT="linux"
 fi
-URL="https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-${OSEXT}.tar.gz"
+ISTIO_BASE_URL=${BASE_URL:-https://github.com/istio/istio/releases/download}
+URL="${ISTIO_BASE_URL}/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-${OSEXT}.tar.gz"
 echo "Downloading $NAME from $URL ..."
 curl -L "$URL" | tar xz
+
+# Current URL for the debian files artifacts. Will be replaced by a proper apt repo.
+DEBURL=http://gcsweb.istio.io/gcs/istio-release/releases/${ISTIO_VERSION}/deb
+curl -L ${DEBURL}/istio-agent-release.deb > istio-${ISTIO_VERSION}/istio-agent-release.deb
+curl -L ${DEBURL}/istio-auth-node-agent-release.deb > istio-${ISTIO_VERSION}/istio-auth-node-agent-release.deb
+curl -L ${DEBURL}/istio-proxy-release.deb > istio-${ISTIO_VERSION}/istio-proxy-release.deb
+
```

Comment:
Sorry, I totally missed this.. downloadIstio is linked to our vanity URL. If anything, can you add a comment to this file to not modify anything except the version ?

Reply:
that too modify by hand.. Say its linked to getIstio.sh